### ISSUE TITLE
Spout Custom Item Support

### ISSUE THREAD

**Issue by FreshGarlic** [CitizensDev/Citizens#595]
At the moment giving an NPC a custom item changes that item to flint. I'm not sure if the Spout Dev team need to add new API for this but it would be awesome if they could hold, trade and store custom items. Again I'm not sure if it's possible yet. Thanks!

**Comment by Farwaykorse** [comment]
Spout uses flint for all custom-items, they just all have a different damage value (like coloured wool).

I don't know where you wanted to use these item. (trader or quester?)

The quester supports them only as a reward. (add the "data:" field to item:)
It doesn't differentiate with flint for an objective or requirement, yet.

**Comment by byklop** [comment]
Hope it will be possible to use "data:" field in those (objective and requirement) soon too. I wanted to make birch log(17:2) delivery quest and it didn't work. But it worked as a reward.

**Comment by AgentK20** [comment]
Agreed. It would be awesome for Citizens to support item data, even if it's just for vanilla items (as it would most likely add support for spout's items).
